packages/database/src/migrations/20200914013333-UpdateLaosSchoolsSchoolIndicatorsEiEFunctioningTVMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
```

Comment:
This migration is for updating an existing overlay `Functioning TV, satellite receiver and dish set`

> 'Functioning TV, satellite receiver and dish set' to 'Functioning TV for teaching and learning purposes'
> (Responses: Yes
> Yes, smart TV
> No
> No data) (Codes: combines codes SchCVD012 and SchCVD012a) (If yes to SchCVD012 but no to SchCVD012a display Yes, if yes to SchCVD012 and yes to SchCVD012a display Yes, smart TV)

Reply:
Nice, thanks for the comments!